docs/src/concrete-playback.md

```diff
@@ -0,0 +1,59 @@
+# Concretely playing back proof harnesses
+
+When the result of a certain check comes back as a `FAILURE`,
+this _experimental_ feature allows users to debug their proof harness and understand what
+caused the failing check.
+
+## Setup
+
+The user needs to have a fairly recent version of the Kani source code somewhere on their computer.
+To do this, run `git clone https://github.com/model-checking/kani.git`.
```

Comment:
Maybe say, this feature require users to declare dependency on the kani crate. BTW, users don't need to do manually clone the repository. Cargo does that for them.

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories

Reply:
Done